### PR TITLE
Move PostgresStrictEncrypt back to its original location

### PR DESCRIPTION
**Pull request by nguyenaiden**
## What
[https://app.zenhub.com/workspaces/db--dw-source-connectors-6333360e0a41155061efbcbd/issues/gh/airbytehq/airbyte/24131](url)

Currently, the source-postgres-strict-encrypt connector is split across 2 different directories:

The tests are run under the source-postgres directory
The docker image is built off of the code in source-postgres-strict-encrypt directory.
The ask here is to consolidate the code and tests for `source-postgres-strict-encrypt` under one directory. This is to prevent confusion and to avoid scenarios where we are not testing the code that is being published.

## How
- Removed the `PostgresSourceRunner`
- Moved back PostgresSourceStrictEncrypt and all the relevant acceptance and integration tests.
##Manual Testing
`Postgres-strict-encrypt` on cloud does not allow disabling of SSL Mode
image

### CODE REVIEW

File: airbyte-integrations/connectors/source-postgres-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/postgres_strict_encrypt/PostgresSourceStrictEncryptAcceptanceTest.java [hunk 1]
```diff
@@ -63,36 +59,36 @@ protected void setupEnvironment(final TestDestinationEnv environment) throws Exc
     environmentVariables.set("DEPLOYMENT_MODE", "CLOUD");
     environmentVariables.set(EnvVariableFeatureFlags.USE_STREAM_CAPABLE_STATE, "true");
     container = new PostgreSQLContainer<>(DockerImageName.parse("postgres:bullseye")
-        .asCompatibleSubstituteFor("postgres"));
+                                              .asCompatibleSubstituteFor("postgres"));
```
[the remark this answers is not in the document]
Comment (akashkulk) [reply airbyte-integrations/connectors/source-postgres-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/postgres_strict_encrypt/PostgresSourceStrictEncryptAcceptanceTest.java]: or you can run `./gradlew :spotlessJavaApply` from the command line

File: airbyte-integrations/connectors/source-postgres-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/postgres_strict_encrypt/PostgresSourceStrictEncryptAcceptanceTest.java [hunk 2]
```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
  */
 
-package io.airbyte.integrations.io.airbyte.integration_tests.sources;
+package io.airbyte.integrations.source.postgres_strict_encrypt;
```
Comment (akashkulk) [inline airbyte-integrations/connectors/source-postgres-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/postgres_strict_encrypt/PostgresSourceStrictEncryptAcceptanceTest.java:5]: nit: Why the different package name structure?

File: airbyte-integrations/connectors/source-postgres/src/test/resources/dummy_config.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (akashkulk) [inline airbyte-integrations/connectors/source-postgres/src/test/resources/dummy_config.json:1]: Why did these files (*.json) need to be added under test/resources in the source-postgres directory? I thought tests needed to be removed from here.

**Review by akashkulk — APPROVED** [review]
Looks good! Just some small nits. 

Out of curiosity, what caused the tests to pass again?

**Comment by nguyenaiden** [comment]
@akashkulk, the tests passed because I wanted to run unit and acceptance tests one at a time on the CI box. The first run passed due to the acceptance tests being empty.

File: airbyte-integrations/connectors/source-postgres-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/postgres_strict_encrypt/PostgresSourceStrictEncryptAcceptanceTest.java (hunk 2 above)
Comment (nguyenaiden) [reply airbyte-integrations/connectors/source-postgres-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/postgres_strict_encrypt/PostgresSourceStrictEncryptAcceptanceTest.java:5]: @akashkulk I followed the same structure that `Mysql` and `Mysql-strict-encrypt` have. The non strict-encrypt variant have their tests under `airbyte.integration_tests.sources`.

Mysql: [https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-mysql/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/MySqlSourceAcceptanceTest.java#L5](url)
Mysql-strict-encrypt: [https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-mysql-strict-encrypt/src/test-integration/java/io/airbyte/integrations/source/mysql_strict_encrypt/MySqlStrictEncryptSourceAcceptanceTest.java#L5](url)

File: airbyte-integrations/connectors/source-alloydb-strict-encrypt/src/main/java/io/airbyte/integrations/source/alloydb/AlloyDbStrictEncryptSource.java [hunk 4]
```diff
@@ -8,7 +8,7 @@
 
 import io.airbyte.integrations.base.IntegrationRunner;
 import io.airbyte.integrations.base.Source;
-import io.airbyte.integrations.source.postgres.PostgresSourceStrictEncrypt;
+import io.airbyte.integrations.source.postgres_strict_encrypt.PostgresSourceStrictEncrypt;
```
Comment (rodireich) [inline airbyte-integrations/connectors/source-alloydb-strict-encrypt/src/main/java/io/airbyte/integrations/source/alloydb/AlloyDbStrictEncryptSource.java:11]: nit: match to source postgres strict package name.
We don't usually use underscores in package names.

**Review by rodireich — APPROVED** [review]

**Comment by nguyenaiden** [comment]
/publish connector=connectors/source-postrgres run-tests=false

**Comment by nguyenaiden** [comment]
/publish connector=connectors/source-postrgres-strict-encrypt run-tests=false

**Comment by nguyenaiden** [comment]
/publish connector=connectors/source-postgres run-tests=false

**Comment by nguyenaiden** [comment]
/publish connector=connectors/source-postgres-strict-encrypt run-tests=false

**Pull request merged**